daml-assistant/integration-tests/src/Main.hs

```diff
@@ -196,9 +196,12 @@ packagingTests tmpDir = testGroup "packaging"
         step "Creating project a..."
         createDirectoryIfMissing True (projectA </> "daml")
         writeFileUTF8 (projectA </> "daml" </> "Main.daml") $ unlines
-            [ "daml 1.2"
+            [ "{-# LANGUAGE EmptyCase #-}"
```

Comment:
Why do we need that now?

Reply:
Because the generic instance derivation apparently creates an empty case. I guess for unit.